mappings/net/minecraft/util/time/Timestamp.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_4316 net/minecraft/util/Timestamp
+CLASS net/minecraft/class_4316 net/minecraft/util/time/Timestamp
```

Comment:
This is more like used by brain? Should be like dynamic stuff, move to `net.minecraft.datafixer.dynamic` if you really want to move this